int16[] node
string cmd
float64 position1
float64 position2
float64 time
int16 count